node-core/services/registry/registry.go

```diff
@@ -167,13 +159,3 @@ func (s *Registry) FetchService(service interface{}) error {
 	}
 	return errUnknownService
 }
-
-func (s *Registry) CommitMultiStore() store.CommitMultiStore {
```

Comment:
not opinionated to Registry

Reply:
yeah we've just moved this function to the Node where it is called from anyways. no need for an extra layer of function calls